docs/tutorial_GPU.ipynb

```diff
@@ -0,0 +1,5769 @@
+{
```

Comment:
<p>Best to use a context manager for file writing</p><pre class="ql-syntax" spellcheck="false">with open('POP_SST.nc', wb) as fp:
    fp.write(file.contents)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocean-eddy-cpt/gcm-filters/pull/69/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>